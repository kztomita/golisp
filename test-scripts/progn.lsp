(print (progn
	 (print "a")
	 (print "b")
	 '1))

(print (progn))

