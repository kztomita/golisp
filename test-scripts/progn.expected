
"a" 
"b" 
1 
nil 
